setuptools/extern/__init__.py

```diff
@@ -71,6 +71,6 @@ def install(self):
 
 names = (
     'packaging', 'pyparsing', 'ordered_set', 'more_itertools', 'importlib_metadata',
-    'zipp', 'importlib_resources', 'jaraco',
+    'zipp', 'importlib_resources', 'jaraco', 'tomli', '_validate_pyproject'
```

Comment:
plz keep the trailing comma:
```suggestion
    'zipp', 'importlib_resources', 'jaraco', 'tomli', '_validate_pyproject',
```